data/deep missions.txt

```diff
@@ -0,0 +1,2317 @@
+			`	"In the field of energy engineering, I'm among the best that the Deep has to offer. I worked on the Dwarf Core." Garrison pauses and chuckles to himself before saying, "I also contributed toward the creation of those comically impractical LP576a battery packs that I have yet to see a single ship using.`
+			`	"Anyhow, because of my past work, the Deep approached me to work on this project. The pay was better than what I was getting before so I accepted it. It was a pleasant surprise to meet my partners as well. Hannah graduated in the same class as me, double majoring in quantum mechanics and ancient history, of all things."`
+			choice
+				`	"What is it that you need from Carbuncle Station that is so secret?"`
+				`	"What about Pierre and Laura?"`
+					goto partners
+			
+			`	"Like I said already, the project is classified. It's not something I would understand, though. It's in Laura's field of expertise. You see, she majored in-"`
+				goto end
+			
+			label partners
+			branch electron
+				has "event: navy using mark ii ships"
+			`	"Pierre is an older guy, he's been working on laser weaponry for the better part of three decades now. Before this he was working on a separate project, but the Deep asked that he take time away from that to work on Project Hawking while they try to find a replacement for him, since the project needed someone with better knowledge in laser weaponry than the rest of us.`
+				goto laura
+			
+			label electron
+			`	"Pierre is an older guy. He has been working on laser weaponry for the better part of three decades now. If you've ever seen an electron beam in action, you have him to thank for that. And a few others, of course, but one of his findings is really what brought the whole project together.`
+			
+			label laura
+			`	"Laura, on the other hand-"`
+			
+			label end
+			`	Before Garrison can finish speaking, the other three scientists return, helping each other carry a sealed metal box. "We're just going to set this in the back, if you don't mind," Pierre says. The scientists make themselves cozy in the bunk rooms and get ready to depart.`
+			`	A chirping from your ship's radio indicates you have received a message from the station. You open it and are not shocked at its contents:`
+			``
+			`		Your immediate departure is required.`
+			`					Carbuncle Station Docking Authority`
+			``
+			`	You signal to your crew and passengers that takeoff is imminent, and obey the wishes of the Lovelace Lab's research station.`
+				launch
+
+
+
+mission "Deep: Project Hawking: Prime"
+	landing
+	invisible
+	source "Prime"
+	to offer
+		has "Deep: Project Hawking: active"
+	to fail
+		has "Deep: Project Hawking: done"
+	npc kill
+		government "Independent"
+		personality waiting
+		ship "Marauder Raven (Engines)" "Krampus"
+	on offer
+		conversation
+			`As you approach the massive <planet> spaceport building, Pierre directs you to park o
```

Comment:
The alloys aren't right in front of the player, so "there" is correct in this case. (Notice how right after this sentence is when the alloys are loaded onto your ship.) I'll change the wording though to avoid any confusion.